### DISCUSSION TITLE
Getting Error when Executing Code

### DISCUSSION THREAD

**Discussion by AshokAWSVM** [conor19w/Binance-Futures-Trading-Bot#47] [Q&A]
Hi Conor,
 Please help me to resolve this issue, when i run the code, getting these error.
 Screenshot attached below
![image](https://github.com/conor19w/Binance-Futures-Trading-Bot/assets/86772562/f257b458-e57b-4d6d-b960-e84a3b12975e)

**Comment by conor19w** [comment]
Hey,
Looks like you're running the bot with python 3.11
The current version of the bot only works with 3.8, I'm working on a new version for 3.11 currently.
For now you should downgrade to 3.8

**Reply by AshokAWSVM** [reply]
Thanks Conor
